config/configgrpc/configgrpc.go

```diff
@@ -150,6 +180,13 @@ type ServerConfig struct {
 	IncludeMetadata bool `mapstructure:"include_metadata"`
 }
 
+// NewDefaultServerConfig() creates a new ServerConfig with any default values set
+func NewDefaultServerConfig() *ServerConfig {
+	return &ServerConfig{
```

Comment:
This function will need to call `configauth.NewDefaultAuthentication` once it exists. We'll need to wait on this PR until that exists.

Reply:
Ohh, does an issue that will create `configauth.NewDefaultAuthentication` exist?

If it does can I get a link

Reply:
Ya it doesn't yet, we need to make it in `configauth` like you are for `configgrpc`.  See https://github.com/open-telemetry/opentelemetry-collector/issues/9821. You can work on that issue as well if you'd like.

Reply:
Just created a PR on this.
Here is the link: https://github.com/open-telemetry/opentelemetry-collector/pull/9825